src/main/java/de/tum/in/www1/exerciseapp/web/rest/CourseResource.java

```diff
@@ -142,4 +141,20 @@ public CourseResource(CourseService courseService) {
         courseService.delete(id);
         return ResponseEntity.ok().headers(HeaderUtil.createEntityDeletionAlert(ENTITY_NAME, id.toString())).build();
     }
+
+
+    /**
+     * GET /user/:courseId/courseResult
+     *
+     * @param courseId the Id of the course
+     * @return collection of Results where the sum of the best result per exercise, for each student in a course is cointained:
```

Comment:
Is it fine to use Result and store in the id the user id?

Reply:
No, this is quite ugly. Please navigate over the participation or introduce a field studentId for sending it over REST (do not store it in the database)

Reply:
done ;) going over participation now